### PR TITLE
Adding unit tests for models

### PR DESCRIPTION
**Pull request by WJXHenry**
**Summary**

Unit tests for some of the model classes.

Added tests:
- User
- MoodEvent
- MoodEventAssociation

Tests to complete:
- FollowableUser
- MoodletUser
- FollowRequest
- FollowList

### CODE REVIEW

File: Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventAssociationTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private String username = "UwU";
+
+    private MoodEventAssociation mockMoodEventAssociation() {
+        return new MoodEventAssociation(moodEvent, username);
+    }
+
+    @Test
+    public void testConstructor() {
+        MoodEventAssociation moodEventAssociation = new MoodEventAssociation(new MoodEvent(EmotionalState.CONFUSED), "Test");
+        assertNotNull(moodEventAssociation);
```
Comment (bojotamara) [inline Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventAssociationTest.java:19]: Might wanna assert that the object actually has CONFUSED and "test" inside of it
Something like
moodEventAssociation.getUsername() == "Test

File: Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+public class MoodEventTest {
+
+    @Mock
+    Bitmap bitmap;
+
+    @Test
+    public void testConstructor() {
+        MoodEvent newMoodEvent = new MoodEvent(EmotionalState.EXCITED);
+        assertNotNull(newMoodEvent);
```
Comment (bojotamara) [inline Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventTest.java:18]: Same thing here, assert that the constructor sets the proper emotional state

File: Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventAssociationTest.java (hunk 1 above)
Comment (WJXHenry) [reply Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventAssociationTest.java:19]: Added constructor content validation tests

File: Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventTest.java (hunk 2 above)
Comment (WJXHenry) [reply Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventTest.java:18]: Added constructor content validation tests

**Review by bojotamara — APPROVED** [review]

**Pull request merged**